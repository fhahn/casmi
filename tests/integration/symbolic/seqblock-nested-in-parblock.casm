
function (symbolic) a : ->Int
function (symbolic) b : ->Int

init main

rule main = {
    a := 1
    {|
        b := 66
        b := 99
    |}
    program(self) := undef
}
