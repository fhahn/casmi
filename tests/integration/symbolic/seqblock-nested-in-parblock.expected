forklog:
tff(symbolNext, type, sym2: $int).
fof(id0,hypothesis,sta(1,sym2)).%CREATE: a
tff(symbolNext, type, sym3: $int).
fof(id1,hypothesis,stb(1,sym3)).%CREATE: b
fof(id2,hypothesis,sta(2,1)).%UPDATE: a
fof(id3,hypothesis,stb(2,99)).%UPDATE: b
fof(final0,hypothesis,sta(0,1)).%FINAL: a
fof(final1,hypothesis,stb(0,99)).%FINAL: b

